src/Authentication/Authentication/Helpers/Errors.cs

```diff
@@ -20,5 +20,8 @@ public static class Errors
         public const string InvokeGraphRequestInputFileNoPathResolvedException = nameof(InvokeGraphRequestInputFileNoPathResolvedException);
         public const string InvokeGraphRequestInputFileNotFilePathException = nameof(InvokeGraphRequestInputFileNotFilePathException);
         public const string InvokeGraphRequestMissingAuthenticationContext = nameof(InvokeGraphRequestMissingAuthenticationContext);
+        public const string InvokeGraphRequestEmptyKeyInJsonString = nameof(InvokeGraphRequestEmptyKeyInJsonString);
```

Comment:
Error constants should ideally go in `ErrorConstants.cs` - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/features/IGRImprovements/src/Authentication/Authentication/ErrorConstants.cs.